runtimes/eden/src/weights/pallet_xcm_benchmarks_fungible.rs

```diff
@@ -0,0 +1,127 @@
+// Copyright 2021 Parity Technologies (UK) Ltd.
+// This file is part of Polkadot.
+
+// Polkadot is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+
+// Polkadot is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU General Public License for more details.
+
+// You should have received a copy of the GNU General Public License
+// along with Polkadot.  If not, see <http://www.gnu.org/licenses/>.
+
+//! Autogenerated weights for `pallet_xcm_benchmarks::fungible`
+//!
+//! THIS FILE WAS AUTO-GENERATED USING THE SUBSTRATE BENCHMARK CLI VERSION 4.0.0-dev
+//! DATE: 2023-03-14, STEPS: `50`, REPEAT: 20, LOW RANGE: `[]`, HIGH RANGE: `[]`
+//! EXECUTION: Some(Wasm), WASM-EXECUTION: Compiled, CHAIN: Some("dev"), DB CACHE: 1024
+
+// Executed Command:
+// ./target/release/nodle-parachain
+// benchmark
+// pallet
+// --chain=dev
+// --steps=50
+// --repeat=20
+// --pallet=pallet_xcm_benchmarks::fungible
+// --extrinsic=*
+// --execution=wasm
+// --wasm-execution=compiled
+// --template=temp.hbs
+// --output=runtimes/eden/src/weights
+
+#![cfg_attr(rustfmt, rustfmt_skip)]
+#![allow(unused_parens)]
+#![allow(unused_imports)]
+
+use frame_support::{traits::Get, weights::Weight};
+use sp_std::marker::PhantomData;
+
+/// Weights for `pallet_xcm_benchmarks::fungible`.
+pub struct WeightInfo<T>(PhantomData<T>);
+impl<T: frame_system::Config> WeightInfo<T> {
+	// Storage: System Account (r:1 w:1)
+	// Storage: System Number (r:1 w:0)
+	// Storage: System ExecutionPhase (r:1 w:0)
+	// Storage: System EventCount (r:1 w:1)
+	// Storage: System Events (r:1 w:1)
+	// Storage: Balances TotalIssuance (r:1 w:1)
+	pub(crate) fn withdraw_asset() -> Weight {
+		Weight::from_ref_time(39_546_000 as u64)
```

Comment:
@Hounsette though we have updated the template, you don't need to recreate the benchmark if you just replace all ` as u64` for numbers with `_u64` or simply remove them.

Reply:
> @Hounsette though we have updated the template, you don't need to recreate the benchmark if you just replace all ` as u64` for numbers with `_u64` or simply remove them

It got out of sync when weight system was updated upstream again I think. Let's update our template and run it again.